sites/feedback/ajax/report-problem-en.html

```diff
@@ -0,0 +1,101 @@
+<details class="brdr-0">
+	<summary class="btn btn-default text-center">Report a problem on this page</summary>
+	<div class="well row">
+		<div class="gc-rprt-prblm">
+			<div class="gc-rprt-prblm-frm gc-rprt-prblm-tggl">
+				<form action="#" id="gc-rprt-prblm-form">
```

Comment:
The form should not make the page reload, it should submit within the page and show the thank you message:
![Screen Shot 2022-05-04 at 12 08 15 PM](https://user-images.githubusercontent.com/4353199/166724081-1e95c3d8-2bb7-4944-80ff-3095a6c35edf.png)

Reply:
Fixed! Thanks